src/components/select/select-list/select-list.component.js

```diff
@@ -59,6 +59,8 @@ const SelectList = React.forwardRef(
       multiColumn,
       tableHeader,
       loaderDataRole,
+      listPlacement = "bottom-start",
+      flipEnabled = true,
```

Comment:
I think it would be better to have the default value `flipEnabled = true` in each of the `Select` components rather than defaulting it in the `SelectList` so that it shows up in the prop tables. Alternatively we could change invert it and surface a `flipDisabled` which would also convey that the flip functionality is "on" by default

![image](https://user-images.githubusercontent.com/44157880/133442581-407b9592-0f28-45cd-aab9-780773f94bbb.png)

![image](https://user-images.githubusercontent.com/44157880/133442738-3cad6b20-652d-455b-bb8d-c45e505b58ec.png)

Reply:
Good point, thanks.

<img width="1042" alt="Zrzut ekranu 2021-09-15 o 16 16 57" src="https://user-images.githubusercontent.com/77274590/133451385-ea0faf67-8438-4d79-9227-e8e22d1b4fd9.png">